process/sandbox-annual-review.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox annual review 
+
+Sandbox projects are subject to an annual review by the TOC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
+
+To keep it lightweight, it is a single phase process - the review doesn't first pass through SIGs and then the TOC. The review document is shared with the community and SIG members can comment on each review, as can any other TOC contributor. The intention is to quickly reach a position where TOC members agree to continue sponsorship. 
+
+## How to file your annual review
+
+CNCF staff will notify the project maintainers when the project review is due. 
+
```

Comment:
It's implicit that projects nearing 12 months since being submitted for sandbox or previous review will be notified. Also projects that are currently in review for incubation will be exempt from annual review. Should a note be added to state the criteria explicitly.

Reply:
The title covers the fact that it’s for Sandbox projects, and that it’s something that should happen annually. The reality is that right now we are well over one year for some projects, and because we have had no process, no notifications have happened yet, so there’s no point being too prescriptive right now over something we can’t immediately fix. It will be CNCF staff’s responsibility to do this in a timely fashion but there needs to be some leeway so everyone can catch up with the backlog. 

I disagree that projects currently in review for incubation should necessarily be exempt from annual review - I think this a matter for discretion for the TOC, CNCF staff and the project to agree amongst themselves (e.g. a project receiving their annual review notification might point at their incubation proposal as a response, and we can take it from there).